### PR TITLE
pc-letter:0.1.0

### PR DESCRIPTION
**Pull request by thatfloflo**
I am submitting
- [ ] a new package
- [x] an update for a package

Description: This update adds several localisation options (`cy`, `es`, `fr`, `de`) and new alignment options for date and reference number fields.

- [X] ensured that my package is licensed such that users can use and distribute the contents of its template directory without restriction, after modifying them through normal use.

### CODE REVIEW

File: packages/preview/pc-letter/0.1.0/README.md [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+
+Dear Reed,
+
+I have recently come across a nice letter template for _Typst_ called `pc-letter`. Since I know that you have a penchant for physical letters, I thought I would use that as an excuse to send a few words your way.
+
+#(letter.valediction)[So long,]
+```
+
+For a more comprehensive example see
+[`example/example.typ`](./example/example.typ).
```
Comment (saecki) [inline packages/preview/pc-letter/0.1.0/README.md]: This file doesn't exist anymore

File: packages/preview/pc-letter/0.1.0/lib.typ [hunk 2]
```diff
@@ -0,0 +1,368 @@
...
+    #set text(
+      size: style.text.size.normal,
+      font: style.text.font,
+      lang: style.locale.lang,
+      region: style.locale.region,
+      discretionary-ligatures: false,
+      features: (onum: 1),
+    )
+    #set par(first-line-indent: 2em)
+    #show heading: set align(style.alignment.headings)
```
Comment (saecki) [inline packages/preview/pc-letter/0.1.0/lib.typ:259]: This line errors when trying to compile the readme example.

File: packages/preview/pc-letter/0.1.0/lib.typ [hunk 3]
```diff
@@ -0,0 +1,368 @@
...
+    #show heading: set block(above: 1em, below: 1em)
+    #show heading.where(level: 1): set text(size: style.text.size.normal, weight: "bold")
+    #show heading.where(level: 2): set text(size: style.text.size.normal, weight: "regular", style: "italic")
+    #let date-y-offset = 0mm
+    #if style.alignment.date-field.y == top {
```
Comment (saecki) [inline packages/preview/pc-letter/0.1.0/lib.typ:264]: This also throws an error with the default arguments.

**Review by saecki — COMMENTED** [review]
Thanks for the update! I've found a broken link in the readme, and there seem to be some issues with the default style configuration.

File: packages/preview/pc-letter/0.1.0/lib.typ (hunk 2 above)
Comment (thatfloflo) [reply packages/preview/pc-letter/0.1.0/lib.typ:259]: Thanks for catching that, seems after fiddling with more options I forgot to go back and test it with the most simple of arguments, namely none. This has been fixed now by making the default `style` arg an empty dictionary so it doesn't end up overwriting actual values with `auto` (:

File: packages/preview/pc-letter/0.1.0/README.md (hunk 1 above)
Comment (thatfloflo) [reply packages/preview/pc-letter/0.1.0/README.md]: Thanks, I had completely overlooked that!

**Comment by thatfloflo** [comment]
Thanks for catching those issues @saecki, I've fixed them and force pushed an amended commit now.

**Comment by saecki** [comment]
Thanks!
